installer/lib/phx_new/generator.ex

```diff
@@ -198,7 +198,7 @@ defmodule Phx.New.Generator do
     config :#{binding[:app_name]}, #{binding[:app_module]}.Repo#{kw_to_config adapter_config[:test]}
     """
 
-    config_inject project_path, "config/prod.secret.exs", """
+    config_inject project_path, "config/runtime.exs", """
```

Comment:
I am afraid this will introduce it it in the wrong place. You need to introduce inside the "if" block and it needs to be indented properly. :)